### ISSUE TITLE
[Feature] Archive from the fullscreen photo view in web

### ISSUE THREAD

**Issue by alextran1502** [immich-app/immich#2297]
### Discussed in https://github.com/immich-app/immich/discussions/2294

<div type='discussions-op-text'>

<sup>Originally posted by **martyfuhry** April 19, 2023</sup>
### The feature

In the fullscreen photo view on web, you should be able to archive the image. This should automatically navigate you to the next image.

![image](https://user-images.githubusercontent.com/100457/233179609-c453fd9d-75c6-49da-a206-977d209cdc03.png)

The above image are the available actions while in fullscreen photo view mode.

![image](https://user-images.githubusercontent.com/100457/233179786-6635bd87-8915-4bdf-842b-94ecaf0e72b0.png)

While multi-selecting a photo, from the Photos main view, the above actions include archiving the selection.

Since this is a discussion, maybe we should standardize the actions available between the two views to the extent that it can be done.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile</div>

**Comment by brighteyed** [comment]
`AssetViewer` is a shared component used by multiple pages. I think the most intuitive behavior for users could be as following:

| Page | Button: Archive/Unarchive/None | Button click action |
|-|-|-|
| Photos | Archive | Archive and navigate to the next asset |
| Archive | Unarchive | Unarchive and navigate to the next asset |
| Favorites or Search results | Archive or unarchive | Archive or unarchive but do not navigate to the next asset. Also reverse toggle button visibility: Archive -> Unarchive and Unarchive -> Archive. Show notification popup |
| Search results | Archive or unarchive | Archive or unarchive but do not navigate to the next asset. Also reverse toggle button visibility: Archive -> Unarchive and Unarchive -> Archive. Show notification popup |
| Album | Archive or Unarchive (if owned by current user) or None (if shared by another user) | Archive or unarchive but do not navigate to the next asset. Also reverse toggle button visibility: Archive -> Unarchive and Unarchive -> Archive. Show notification popup |

**Comment by jon-f-novastor** [comment]
I agree that standardizing the functions in the toolbar in the various sections of the web + mobile app would be great to have. Thanks.
